scripts/3.TCGA-MLexample_Pathway.py

```diff
@@ -463,25 +441,25 @@ def get_threshold_metrics(y_true, y_pred, tissue='all'):
 
 # ## Investigate the predictions
 
-# In[44]:
+# In[42]:
 
 predict_df = pd.DataFrame.from_items([
     ('sample_id', X_sub.index),
     ('testing', X_sub.index.isin(X_test.index).astype(int)),
     ('status', y_sub),
-    ('decision_function', pipeline.decision_function(X_sub)),
-    ('probability', pipeline.predict_proba(X_sub)[:, 1]),
+    ('decision_function', cv_pipeline.decision_function(X_sub)),
```

Comment:
the `cv_pipeline` object is prefit with the best hyperparameters found with the optimal hyperparameters?